submissions/bilkisuTace/HTML/product.html

```diff
@@ -7,6 +7,10 @@
 <body>
 	<header>	
 		<h1>LADIES & GENTS... <small><i>there's a shoe for everyone.</i></small></h1>
+		<ul>
+			<li><a href="../html-forms/contact.html"><h3>Contact Page</h3></a></li>
```

Comment:
There is an oncoming task to link your pages. You should remove this for now @bilkisutace